server/services/omnichannel-voip/service.ts

```diff
@@ -1,14 +1,24 @@
 import { Db } from 'mongodb';
+import { Meteor } from 'meteor/meteor';
+import { Match, check } from 'meteor/check';
+import { Random } from 'meteor/random';
```

Comment:
Pls don't use meteor on services 😬 that makes the services dependent on the rest of the platform and difficult for them to be "independent"

If you must validate the data types, use an `assert`, or you can do what other services do of trusting that the function will be called with the right params from the endpoint handler 👀

Reply:
I shall assume that all is good. :)